src/syntax/syntax.rkt

```diff
@@ -42,22 +42,25 @@
            variable?))
 
 (module+ test
-  (require rackunit))
+  (require rackunit
+           rival))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Real operators
 ;; Pure mathematical operations
 
+;; TODO: specs should really be associated with impls
+;; unfortunately Herbie still mandates that every impl
+;; has an associated operator so the spec is here
+
 ;; A real operator requires
 ;;  - a (unique) name
 ;;  - input and output types
-;;  - exactly one of:
-;;     - a Rival interval implementation
-;;     - a specification ["accelerator"]
-;;  - deprecated? flag [#f by default]
+;;  - optionally a specification [#f by default]
```

Comment:
Wait how can a specification be optional? What does an operator without a specification even mean?

Reply:
I had a comment about this somewhere. Basically, I think the eventual design is that _operator implementations_ have specifications, but we still require that every operator implementation _implements_ an operator. So the spec information is awkwardly associated with the operator rather than the implementation. I think your other comments might be related to this weirdness. The goal is to move to that design somehow